tests/unit/test_alteration.py

```diff
@@ -136,3 +136,162 @@ def test_validate_invalid_share_structure_alteration():
     print(errors)
 
     assert not is_valid
+
+
+def test_validate_invalid_court_order_file_number_min():
```

Comment:
Do you think you could combine the invalid court_order test cases together in a @pytest.mark.parametrize sort of thing? These are good tests but they double the size of the alteration test file for one field in the alteration schema

Reply:
Sure. Will do it. Thanks!

Reply:
I did that using @pytest.mark.parametrize and removed some tests for valid values that were tested already on the example data. Looks better now. Thanks!